cmd/lk/ingress.go

```diff
@@ -32,6 +32,73 @@ const ingressCategory = "Ingress"
 var (
 	IngressCommands = []*cli.Command{
 		{
+			Name:     "ingress",
+			Usage:    "Import outside media sources into a LiveKit room",
+			Category: "I/O",
+			Commands: []*cli.Command{
+				{
+					Name:   "create",
+					Usage:  "Create an ingress",
+					Before: createIngressClient,
+					Action: createIngress,
+					Flags: withDefaultFlags(
```

Comment:
`withDefaultFlags` is no longer necessary for `urfave/cli/v3`. We can now move common flag to the root command and set `Persistent: true` on these flags. They should then be available on all subcommands.

Reply:
Done